rllib/evaluation/sampler.py

```diff
@@ -856,7 +831,11 @@ def _process_observations(
             # This will be filled with dummy observations below.
             all_agents_obs = {}
 
-        if not is_new_episode:
+        # If this episode is brand-new, call the episode start callback(s).
```

Comment:
Needed to add this flag here to the old `Episode` class. Otherwise, we would never start increasing the `length` property. This is not a problem for EpisodeV2.